[2023-08-08T21:41:14.938-0300] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Investimento_api.Extracao_Invest manual__2023-08-09T00:40:23.540472+00:00 [queued]>
[2023-08-08T21:41:15.019-0300] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Investimento_api.Extracao_Invest manual__2023-08-09T00:40:23.540472+00:00 [queued]>
[2023-08-08T21:41:15.019-0300] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-08T21:41:15.140-0300] {taskinstance.py:1327} INFO - Executing <Task(SparkSubmitOperator): Extracao_Invest> on 2023-08-09 00:40:23.540472+00:00
[2023-08-08T21:41:15.142-0300] {standard_task_runner.py:57} INFO - Started process 17886 to run task
[2023-08-08T21:41:15.145-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Investimento_api', 'Extracao_Invest', 'manual__2023-08-09T00:40:23.540472+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpu4mx258p']
[2023-08-08T21:41:15.146-0300] {standard_task_runner.py:85} INFO - Job 11: Subtask Extracao_Invest
[2023-08-08T21:41:15.336-0300] {task_command.py:410} INFO - Running <TaskInstance: Investimento_api.Extracao_Invest manual__2023-08-09T00:40:23.540472+00:00 [running]> on host VMEnricolm.myguest.virtualbox.org
[2023-08-08T21:41:15.668-0300] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Investimento_api' AIRFLOW_CTX_TASK_ID='Extracao_Invest' AIRFLOW_CTX_EXECUTION_DATE='2023-08-09T00:40:23.540472+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-09T00:40:23.540472+00:00'
[2023-08-08T21:41:15.700-0300] {base.py:73} INFO - Using connection ID 'spark_default' for task execution.
[2023-08-08T21:41:15.701-0300] {spark_submit.py:340} INFO - Spark-Submit cmd: spark-submit --master yarn --name Extracao_Invest --queue root.default /home/enricolm/Documents/Sprinklr_Airflow/dadosvm/Airflow_Investimento/Scripts/Operators/Raw_Operator.py
[2023-08-08T21:42:11.511-0300] {spark_submit.py:491} INFO - 23/08/08 21:42:11 WARN Utils: Your hostname, VMEnricolm resolves to a loopback address: 127.0.1.1; using 192.168.0.63 instead (on interface enp0s3)
[2023-08-08T21:42:11.537-0300] {spark_submit.py:491} INFO - 23/08/08 21:42:11 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2023-08-08T21:42:16.921-0300] {spark_submit.py:491} INFO - Exception in thread "main" org.apache.spark.SparkException: When running with master 'yarn' either HADOOP_CONF_DIR or YARN_CONF_DIR must be set in the environment.
[2023-08-08T21:42:16.921-0300] {spark_submit.py:491} INFO - at org.apache.spark.deploy.SparkSubmitArguments.error(SparkSubmitArguments.scala:650)
[2023-08-08T21:42:16.921-0300] {spark_submit.py:491} INFO - at org.apache.spark.deploy.SparkSubmitArguments.validateSubmitArguments(SparkSubmitArguments.scala:281)
[2023-08-08T21:42:16.921-0300] {spark_submit.py:491} INFO - at org.apache.spark.deploy.SparkSubmitArguments.validateArguments(SparkSubmitArguments.scala:237)
[2023-08-08T21:42:16.921-0300] {spark_submit.py:491} INFO - at org.apache.spark.deploy.SparkSubmitArguments.<init>(SparkSubmitArguments.scala:122)
[2023-08-08T21:42:16.921-0300] {spark_submit.py:491} INFO - at org.apache.spark.deploy.SparkSubmit$$anon$2$$anon$3.<init>(SparkSubmit.scala:1094)
[2023-08-08T21:42:16.921-0300] {spark_submit.py:491} INFO - at org.apache.spark.deploy.SparkSubmit$$anon$2.parseArguments(SparkSubmit.scala:1094)
[2023-08-08T21:42:16.921-0300] {spark_submit.py:491} INFO - at org.apache.spark.deploy.SparkSubmit.doSubmit(SparkSubmit.scala:86)
[2023-08-08T21:42:16.922-0300] {spark_submit.py:491} INFO - at org.apache.spark.deploy.SparkSubmit$$anon$2.doSubmit(SparkSubmit.scala:1111)
[2023-08-08T21:42:16.922-0300] {spark_submit.py:491} INFO - at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:1120)
[2023-08-08T21:42:16.922-0300] {spark_submit.py:491} INFO - at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
[2023-08-08T21:42:16.966-0300] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/enricolm/Documents/Sprinklr_Airflow/dadosvm/Airflow_Investimento/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 156, in execute
    self._hook.submit(self._application)
  File "/home/enricolm/Documents/Sprinklr_Airflow/dadosvm/Airflow_Investimento/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 422, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master yarn --name Extracao_Invest --queue root.default /home/enricolm/Documents/Sprinklr_Airflow/dadosvm/Airflow_Investimento/Scripts/Operators/Raw_Operator.py. Error code is: 1.
[2023-08-08T21:42:17.030-0300] {taskinstance.py:1345} INFO - Marking task as FAILED. dag_id=Investimento_api, task_id=Extracao_Invest, execution_date=20230809T004023, start_date=20230809T004114, end_date=20230809T004217
[2023-08-08T21:42:17.353-0300] {standard_task_runner.py:104} ERROR - Failed to execute job 11 for task Extracao_Invest (Cannot execute: spark-submit --master yarn --name Extracao_Invest --queue root.default /home/enricolm/Documents/Sprinklr_Airflow/dadosvm/Airflow_Investimento/Scripts/Operators/Raw_Operator.py. Error code is: 1.; 17886)
[2023-08-08T21:42:17.618-0300] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-08-08T21:42:17.633-0300] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
